spec/motion-spec/mac_bacon_spec.rb

```diff
@@ -194,7 +194,7 @@ def triggerChange
         @observable.an_attribute = 'changed'
       end
 
-      behaves_like 'waiting in before/after filters'
+      include_examples 'waiting in before/after filters'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.